ql/ql/src/codeql_ql/style/DeadCodeQuery.qll

```diff
@@ -208,8 +210,9 @@ private AstNode benign() {
   result instanceof Comment or
   not exists(result.toString()) or // <- invalid code
   // cached-stages pattern
-  result.(Module).getAMember().(ClasslessPredicate).getName() = "forceStage" or
-  result.(ClasslessPredicate).getName() = "forceStage" or
+  result.(Module).getAMember().(ClasslessPredicate).getName() =
+    ["forceStage", "forceCachingInSameStageforceCachingInSameStage"] or
```

Comment:
I feel like I'm seeing double. Should this be simply `forceCachingInSameStage`?

Reply:
And the case was also redundant.  
The below case in the disjunction already covered it.